README.md

```diff
@@ -1,22 +1,22 @@
-# Emby for Kodi
+# Jellyfin for Kodi
 
-[![EmbyKodi_Banner](https://i.imgur.com/hx4cx41.png)](https://emby.media/community/index.php?/forum/99-kodi/)
+[![EmbyKodi_Banner](https://i.imgur.com/hx4cx41.png)](https://forum.jellyfin.org/)
```

Comment:
What is `EmbyKodi_Banner` and can we point it to Jellyfin-branded picture? Preferably served from github instead of imgur...